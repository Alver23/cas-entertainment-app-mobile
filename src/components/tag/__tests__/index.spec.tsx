// Dependencies
import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { Text } from 'react-native-elements';

import Tag from '../index';

describe('<Tag />', () => {
  let component: ShallowWrapper;

  const props = {
    text: 'placeholder',
  };

  beforeEach(() => {
    component = shallow(<Tag {...props} />);
  });

  afterEach(() => {
    jest.clearAllMocks();
  });

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });

  it('should call the onChangeText void', () => {
    expect(component.find(Text).exists()).toBeTruthy();
  });
});
