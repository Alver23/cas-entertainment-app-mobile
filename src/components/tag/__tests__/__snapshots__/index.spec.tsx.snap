// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tag /> should render correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#2089dc",
      "borderRadius": 8,
      "paddingHorizontal": 8,
      "paddingVertical": 4,
    }
  }
>
  <Themed.Text>
    placeholder
  </Themed.Text>
</View>
`;
